src/taipy/gui/gui.pyi

```diff
@@ -37,6 +37,7 @@ class Gui:
     __DO_NOT_UPDATE_VALUE: Incomplete
     __RE_HTML: Incomplete
     __RE_MD: Incomplete
+    __RE_PY: Incomplete
```

Comment:
Dr Irv had propsed a change about this Incomplete  (change the import to from typing import Any as Incomplete)
Do you have any take on this ?
I guess this is generated and cannot be changed ?

Reply:
we cannot change this as it is generated by stubgen. we can override this manually but I dont see the point of us doing it